include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -1,6 +1,20 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+/* How to build the Doxygen HTML pages:
+
++ Install Doxygen (https://www.doxygen.nl/download.html)
++ Running from the command line:
+  + cd to (Repository Root)\docs
+  + Run "\Program Files\doxygen\bin\doxygen.exe" Doxyfile
```

Comment:
this is Windows specific

Reply:
Whoops, I added 'for windows' at the top.